packages/terra-tabs/src/common-tabs/_Tab.jsx

```diff
@@ -154,14 +172,39 @@ const Tab = ({
 
   if (isIconOnly) {
     attributes['aria-label'] = label;
+    if (isClosable) {
+      attributes['aria-label'] = `${label} ${tabDeleteLabel}`;
+    }
   }
 
+  function onCloseClick(event) {
+    event.stopPropagation();
+    onClosingTab(itemKey, metaData, event);
```

Comment:
I think you should not trigger this for disabled tab !! User can delete the Disabled tab with mouse click as

Reply:
Done https://github.com/cerner/terra-framework/commit/628052153b88461ef88733be6b05c1fb519e3f41